IdentityCore/src/validation/MSIDAuthority.h

```diff
@@ -102,6 +102,8 @@ typedef void(^MSIDOpenIdConfigurationInfoBlock)(MSIDOpenIdProviderMetadata * _Nu
 - (void)loadOpenIdMetadataWithContext:(nullable id<MSIDRequestContext>)context
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->